docker.sh

```diff
@@ -24,13 +24,8 @@ docker run -d \
   --mount type=bind,source="$(pwd)"/"$CONF_PATH",target=/app/instance/app.conf \
   $IMAGE_NAME
 
-green=`tput setaf 2`
-reset=`tput sgr0`
```

Comment:
Consider 
```
green=`tput setaf 2 || echo ''`
reset=`tput sgr0 || echo ''`
```
... though it'd be nice to understand why it won't work, but don't let that block the PR.

Reply:
Thanks, I'll add it back in.